test/js/pageobjects/CrossDeviceIntro.js

```diff
@@ -0,0 +1,36 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceIntro extends Base{
+
+  get crossDeviceIntroTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get smsIcon() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stageIcon-sms')}
+  get takePhotosIcon() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stageIcon-take-photos')}
+  get returnToComputerIcon() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stageIcon-return-computer')}
+  get smsMessage() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stage:nth-child(1) > .onfido-sdk-ui-crossDevice-Intro-stageMessage')}
+  get takePhotosMessage() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stage:nth-child(2) > .onfido-sdk-ui-crossDevice-Intro-stageMessage')}
+  get returnToComputerMessage() { return this.$('.onfido-sdk-ui-crossDevice-Intro-stage:nth-child(3) > .onfido-sdk-ui-crossDevice-Intro-stageMessage')}
```

Comment:
Code can be updated to include a more specific class name for these list items so that you don't have to use the `nth-child` selector for lines 10-12.